fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -228,7 +227,7 @@ protected RdfStream getResourceTriples() {
     protected RdfStream getResourceTriples(final int limit) {
         // use the thing described, not the description, for the subject of descriptive triples
         if (resource() instanceof NonRdfSourceDescription) {
```

Comment:
In principle, you are correct. In practice, I am unable to get the unit tests to pass. 

I am also concerned about this line in a more general sense: why is `resource` being _redefined_ in a method called `getResourceTriples`? That seems entirely wrong to me: any sort of accessor function should not be redefining internal, private variables. To me, this points to a need to refactor some of this code, and not just this particular `instanceof` check.

That is to say, I am leaving this as it stands for now.